pages/common/inkmake.md

```diff
@@ -0,0 +1,24 @@
+# inkmake
+
+> GNU Makefile-style SVG exporting using Inkscape's backend.
+> More information: <https://github.com/wader/inkmake>.
+
+- Display help:
```

Comment:
One more thing: @yutyo will you please move this example to the bottom? We generally put the simplest usage of the command (in this case the second example, `inkmake {{inkfile}}`) at the top, working our way down with more complex examples, and finally if there's still room, pointing to extra help via `--help`.

Reply:
> One more thing: @yutyo will you please move this example to the bottom? We generally put the simplest usage of the command (in this case the second example, `inkmake {{inkfile}}`) at the top, working our way down with more complex examples, and finally if there's still room, pointing to extra help via `--help`.

Done.